GPU/Common/SplineCommon.cpp

```diff
@@ -926,16 +926,19 @@ void DrawEngineCommon::SubmitSpline(const void *control_points, const void *indi
 	patch.patchFacing = patchFacing;
 
 	if (g_Config.bHardwareTessellation && g_Config.bHardwareTransform && !g_Config.bSoftwareRendering) {
-		float *pos = (float*)(decoded + 65536 * 18); // Size 3 float
-		float *tex = pos + count_u * count_v * 3; // Size 3 float
-		float *col = tex + count_u * count_v * 3; // Size 4 float
+		int stride = g_Config.iGPUBackend == GPU_BACKEND_VULKAN || g_Config.iGPUBackend == GPU_BACKEND_DIRECT3D11 ? 4 : 3;
```

Comment:
Hmmmm, difficult conversation... I don't know what are you talking about well.
Sorry for my poor English skill.
I belive this line is right but I agree that it's a bit not readable.

`int stride = g_Config.iGPUBackend == GPU_BACKEND_VULKAN || g_Config.iGPUBackend == GPU_BACKEND_DIRECT3D11 ? 4 : 3;`

is same as

```
int stride;
if (g_Config.iGPUBackend == GPU_BACKEND_VULKAN || g_Config.iGPUBackend == GPU_BACKEND_DIRECT3D11)
  stride = 4;
else
  stride = 3;
```

Yeah, however someone or some compilers might misunderstand.



I think probably @hrydgard is saying that means it needs parenthesis for expression like this.

`stride = (g_Config.iGPUBackend == GPU_BACKEND_VULKAN || g_Config.iGPUBackend == GPU_BACKEND_DIRECT3D11) ? 4 : 3;`

or just simply

`stride = 4`

Right?

I think all backends fix to 4 is better simple solution.